spec/factories/notes.rb

```diff
@@ -0,0 +1,8 @@
+FactoryBot.define do
+  factory :note do
+    title { Faker::Book.title }
+    content { Faker::Lorem.paragraphs(number: 3).join("\n") }
+    type { 1 }
```

Comment:
Ya que `type` es un enum te recomiendo la siguiente lectura para entender como funciona:
https://blog.saeloun.com/2022/01/05/how-to-use-enums-in-rails/

Después te recomiendo ver que hace este método:
https://apidock.com/ruby/Hash/values
y este método:
https://apidock.com/ruby/Array/sample

Con eso debería alcanzar para que puedas crear notas con `type` aleatorios usando la factory del modelo.

Reply:
Gracias! Leí la doc e iba a hacer algo así:
```suggestion
    type { { review: 0, critique: 1 }.values.sample }
```

Pero después vi que puedo samplearlo directamente desde el modelo y quedaría así:
```suggestion
    type { Note.types.keys.sample }
```

Creo que se ve mejor

Reply:
Gracias! ahí leí la doc y lo use ambos métodos.